thoth/user_api/openapi_server.py

```diff
@@ -40,6 +40,7 @@
 from thoth.storages import __version__ as __storages__version__
 from thoth.python import __version__ as __python__version__
 from thoth.messaging import __version__ as __messaging__version__
+from thoth.messaging import producer
```

Comment:
Looks like this import is causing issues on new rollouts:

https://github.com/thoth-station/user-api/issues/1458

```suggestion
from thoth.messaging.producer import producer
```